lib/modules/datasource/conda/types.ts

```diff
@@ -72,20 +69,4 @@ export interface CondaFile {
   version: CondaVersion;
   md5: string;
   type: string;
-  attrs: {
```

Comment:
Why are all these removed?

Reply:
they are not used at all and the property is not stable